src/status_im/contexts/communities/actions/share_community_channel/view.cljs

```diff
@@ -11,9 +11,16 @@
 (defn view
   []
   (fn []
-    (let [{:keys [url chat-id]}           (rf/sub [:get-screen-params])
+    (let [params                          (rf/sub [:get-screen-params])
+          ;; NOTE(seanstrom): We need to store these screen params for when the modal closes
+          ;; because the screen params will be cleared.
+          {:keys [url chat-id]}           @(rn/use-ref-atom params)
           {:keys [color emoji chat-name]} (rf/sub [:chats/community-channel-ui-details-by-id chat-id])
-          window-width                    (rf/sub [:dimensions/window-width])]
+          on-share-community-channel      (rn/use-callback
+                                           #(rf/dispatch
+                                             [:communities/share-community-channel-url-with-data
```

Comment:
There is a navigate back somewhere in this component. That can be extracted to the let bind and wrapped in a `use-callback`.

Not related to your code, but a simple housekeeping win.